Exercise/Destroyer/destroyer.js

```diff
@@ -0,0 +1,13 @@
+function destroyer(arr) {
+    var args = arguments;
+    return arr.filter(function(el) {
+        return numToDestroy(args,el);
+    })
+}
+
+function numToDestroy(args,valueToDestroy){
```

Comment:
Nie powinieneś nazywać argumentu dla tej funkcji `args`. Wszystkie nazwy funkcji, argumenty, parametry, zmienne powinny być semantyczne. W tym przypadku nie wiem czy to ma być array czy obiekt więc lepiej to nazwać `arr` albo `arrayOfNumbers`.